edx_notifications/stores/sql/tests/test_store_provider.py

```diff
@@ -1126,12 +1130,14 @@ def test_get_all_user_preferences_with_name(self):
         Test all user preferences with name
         """
         user_preference1 = self._save_user_notification_preference(
+            number_of_queries=3,
```

Comment:
Can you include some comments here describing why the first case expects 3 queries and the second case expects 2 queries